src/Microsoft.AspNet.OData.Shared/ExpressionHelperMethods.cs

```diff
@@ -28,6 +28,7 @@ internal class ExpressionHelperMethods
         private static MethodInfo _skipMethod = GenericMethodOf(_ => Queryable.Skip<int>(default(IQueryable<int>), default(int)));
         private static MethodInfo _enumerableSkipMethod = GenericMethodOf(_ => Enumerable.Skip<int>(default(IEnumerable<int>), default(int)));
         private static MethodInfo _whereMethod = GenericMethodOf(_ => Queryable.Where<int>(default(IQueryable<int>), default(Expression<Func<int, bool>>)));
+        private static MethodInfo _enumerableContainsMethod = GenericMethodOf(_ => Enumerable.Contains<int>(default(IQueryable<int>), default(int)));
```

Comment:
do you think we should add similar for Queryable.Contains

Reply:
Added. This will be in the next commit.